### PR TITLE
Images

### PR DESCRIPTION
**Pull request by alexholdenmiller**
Tentative PR for adding image paths to DialogData. Should it be a different class? Maybe not, this isn't that much more complicated.

### CODE REVIEW

File: parlai/core/build_data.py [hunk 1]
```diff
@@ -23,7 +23,6 @@ def download(path, url, redownload=True):
     download tar file again if it is present (default true).
     """
     if redownload or not os.path.isfile(path):
-        # only download if file hasn't been downloaded already
         filename = wget.download(url, out=path)
```
Comment (jaseweston) [inline parlai/core/build_data.py]: why is the comment gone?

File: parlai/core/dialog_teacher.py [hunk 2]
```diff
@@ -153,12 +153,14 @@ class DialogData(object):
...
+    - x is a query and possibly context
     ... can contain additional fields, specifically
-        y is an iterable of label(s) for that query
-        r is the str reward for getting that query correct
-        c is an iterable of label candidates that the student can choose from
-    new_episode? is a boolean value specifying whether that example is the start
+      - y is an iterable of label(s) for that query
+      - r is the str reward for getting that query correct
+      - c is an iterable of label candidates that the student can choose from
+      - i is a str path to an image on disk, which will be loaded by the data
```
Comment (jaseweston) [inline parlai/core/dialog_teacher.py]: what about processed images?

File: parlai/core/build_data.py (hunk 1 above)
Comment (alexholdenmiller) [inline parlai/core/build_data.py]: I figured the function docstring explained it better

File: parlai/core/dialog_teacher.py (hunk 2 above)
Comment (alexholdenmiller) [inline parlai/core/dialog_teacher.py]: yeah need to confirm exactly how we want to do this, but the basic solution I set up in load_data was that a folder should exist in the same directory as the images which contains the name of the preprocessor ('vgg') which contains a preprocessed version of the images

see load_data below for that

**Pull request merged**